Snowflake.Data/Core/Tools/FileOperations.cs

```diff
@@ -2,17 +2,34 @@
  * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
  */
 
+using System;
 using System.IO;
+using System.Linq;
 
 namespace Snowflake.Data.Core.Tools
 {
+    using System.Runtime.InteropServices;
```

Comment:
I would prefer to move it before namespace

Reply:
Done